spec/open-api.yml

```diff
@@ -818,6 +818,20 @@ components:
             }
           }
         }
+    AggregatorPipelinesMetrics:
+      type: object
+      description: Map of metrics/error for each pipeline ID.
+      additionalProperties:
+        type: object
+        properties:
+          data:
+            nullable: true
+            $ref: "#/components/schemas/AgentMetrics"
```

Comment:
I know the name `AgentMetrics` looks off. But we are reusing the same type.
We should rename it, but not now.